pages/common/ftp.md

```diff
@@ -0,0 +1,23 @@
+# ftp
+
+> Internet File Transfer Program
+
+- Connect to a FTP server:
+
+`ftp {{url}}`
+
+- Switch to binary transfer mode (graphics, compressed files etc):
+
+`binary`
+
+- Turn off prompts for each process:
+
+`prompt off`
+
+- Download multiple files:
+
+`mget {{glob_expression}}`
```

Comment:
Maybe move "glob expression" to the description, and use an actual example in the token.